[2023-05-01T17:44:28.565+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:27.322527+00:00 [queued]>
[2023-05-01T17:44:28.576+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:27.322527+00:00 [queued]>
[2023-05-01T17:44:28.577+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:44:28.577+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-01T17:44:28.577+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:44:28.596+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-05-01 17:44:27.322527+00:00
[2023-05-01T17:44:28.603+0000] {standard_task_runner.py:55} INFO - Started process 18616 to run task
[2023-05-01T17:44:28.606+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-05-01T17:44:27.322527+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/processing_user.py', '--cfg-path', '/tmp/tmp7of1cd1g']
[2023-05-01T17:44:28.608+0000] {standard_task_runner.py:83} INFO - Job 58: Subtask is_api_available
[2023-05-01T17:44:28.680+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:27.322527+00:00 [running]> on host 9bd2cedcb1a8
[2023-05-01T17:44:28.765+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T17:44:27.322527+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-01T17:44:27.322527+00:00
[2023-05-01T17:44:28.767+0000] {http.py:120} INFO - Poking: api/
[2023-05-01T17:44:28.784+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-05-01T17:44:28.786+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-05-01T17:44:29.570+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-05-01T17:44:29.586+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230501T174427, start_date=20230501T174428, end_date=20230501T174429
[2023-05-01T17:44:29.621+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-01T17:44:29.645+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
